src/FormControl.vue

```diff
@@ -23,6 +21,8 @@ export default {
       this.$set(this.values, name, value);
     },
     submit() {
+      const isValid = this.$refs.observer.validate()
```

Comment:
```suggestion
    async submit() {
      const isValid = await this.$refs.observer.validate()
```
According to https://logaretm.github.io/vee-validate/api/validation-observer.html validate function returns promise.